### ISSUE TITLE
looking for maintainer

### ISSUE THREAD

**Issue by stef-levesque** [stef-levesque/vscode-shader#70]
Hey @Argmaster let's discuss here if you're interested

**Comment by Argmaster** [comment]
Well, I can't promise very active development, but I could try squishing basic repository maintainance and some bugfixing / extending somewhere into my schedule, along with other project I am currently handling, as my interests lie somewhere around GLSL and graphics APIs. It's just about keeping this project floating on the surface and at least giving the community chance to extend it. I never adopted a project before, so I'm not sure how to approach it.

**Comment by stef-levesque** [comment]
What is currently problematic with this project? I have local changes to update the dependencies and fix the ripgrep, but wonder if there another urgent fix to add before I push another release

**Comment by Argmaster** [comment]
No, not necessarily, I think It will be enough just to notify the world that this repo is alive. In general I was bothering you because I have noticed that this repository seemed dead while it is a great project. I figured the sooner I try to reach you the higher chance you'll respond and there will be a better chance that future work done to extend this project won't be wasted.

For my purposes I'd love to have GLSL support at some point and I have noticed there is some PR about it, but haven't reviewed it deeper than just by scrolling it top down and there is no reason to force diving into this topic right now.

**Comment by stef-levesque** [comment]
I'll look at #62 and #67 then
